libraries/adaptive-expressions/src/builtinFunctions/divide.ts

```diff
@@ -26,7 +26,11 @@ export class Divide extends MultivariateNumericEvaluator {
      * @private
      */
     private static func(args: any[]): number {
-        return Math.floor(Number(args[0]) / Number(args[1]));
+        let result: number = Number(args[0]) / Number(args[1]);
```

Comment:
`let` -> `const`

Reply:
Ah my bad, I was trying to follow convention I saw in other files, but I see that 'let' was primarily only used for actual mutable vars. I see const other places accordingly. pushing the change now